Change the env variable for the default include path, 
to PIC30_CPATH from CPATH since the include path for pic30-gcc
is of course different from the one for native GCC.

--- ./gcc/c-incpath.c.~1~	2009-02-01 21:20:37.000000000 +0100
+++ ./gcc/c-incpath.c	2009-02-02 04:33:45.000000000 +0100
@@ -380,7 +380,7 @@
 
   /* CPATH and language-dependent environment variables may add to the
      include chain.  */
-  add_env_var_paths ("CPATH", BRACKET);
+  add_env_var_paths ("PIC30_CPATH", BRACKET);
   add_env_var_paths (lang_env_vars[idx], SYSTEM);
   
   target_c_incpath.extra_pre_includes (sysroot, iprefix, stdinc);
